apps/coordinator/src/tests/signing.jsx

```diff
@@ -30,12 +31,70 @@ class SignMultisigTransactionTest extends Test {
     if (this.params.keystore === HERMIT) {
       tempResult = this.interaction().parse(result);
     }
-    return tempResult.signatures ? tempResult.signatures : tempResult;
+
+    const finalResult = tempResult.signatures
+      ? tempResult.signatures
+      : tempResult;
+
+    return finalResult;
   }
 
   // eslint-disable-next-line class-methods-use-this
   matches(expected, actual) {
-    return JSON.stringify(expected) === JSON.stringify(actual);
+    // If either is null/undefined or lengths don't match, fail immediately
+    if (!expected || !actual || expected.length !== actual.length) {
+      return false;
+    }
+
+    // Get the unsigned PSBT for validation
+    let psbtHex;
+    try {
+      const unsignedPSBTWrapper = unsignedMultisigPSBT(
+        this.params.network,
+        this.params.inputs,
+        this.params.outputs,
+      );
+
+      // The unsignedMultisigPSBT returns { data, txn } where data is the actual PSBT
+      psbtHex = unsignedPSBTWrapper.data.toBuffer().toString("hex");
```

Comment:
Even better I think there's a toHex method

```suggestion
      psbtHex = unsignedPSBTWrapper.data.toHex();
```

Reply:
fixed.